src/golang/lib/job/databricks.go

```diff
@@ -171,3 +110,117 @@ func (j *DatabricksJobManager) EditCronJob(ctx context.Context, name string, cro
 func (j *DatabricksJobManager) DeleteCronJob(ctx context.Context, name string) JobError {
 	return nil
 }
+
+func (j *DatabricksJobManager) CreateTask(
+	ctx context.Context,
+	spec Spec,
+	parentOperatorNames []string,
+) (*jobs.JobTaskSettings, error) {
+
+	// Get the entrypoint file for Databricks.
+	storageConfig, err := spec.GetStorageConfig()
+	if err != nil {
+		return nil, errors.Wrap(err, "Spec unexpectedly has no storage config.")
+	}
+	scriptFile, specStr, err := j.mapJobTypeToFile(spec)
+	if err != nil {
+		return nil, err
+	}
+	bucket := storageConfig.S3Config.Bucket
```

Comment:
Do we ensure that databricks can only be used when the metadata store is S3, so we don't need to explicitly check here?

Reply:
https://github.com/aqueducthq/aqueduct/blob/c56eb2e89b76d8064436fcefec4f7f0c6e9134ef/src/golang/lib/workflow/operator/config.go#L131 we do that here.